src/psyclone/tests/dynamo0p3_stubgen_test.py

```diff
@@ -0,0 +1,775 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
```

Comment:
pylint errors:

C:684, 0: Line too long (80/79) (line-too-long)
W: 78, 4: Unused variable 'ModuleGen' (unused-variable)
W:122, 4: Unused variable 'module' (unused-variable)
W:148, 4: Unused variable 'module' (unused-variable)

Reply:
Thanks for spotting - all fixed.